src/components/Data/Collections/Browse.vue

```diff
@@ -1,13 +1,135 @@
 <template>
   <div class="wrapper">
-    <crudl-document :index="$route.params.index" :collection="$route.params.collection">
-      <document-item></document-item>
+    <headline>
+      {{$route.params.collection}} - Browse
```

Comment:
be carefull about usage of `this.$route.params` please use `collection` & `index` from properties